frontend/src/components/translation/TranslationTable.tsx

```diff
@@ -60,10 +62,12 @@ const TranslationTable = ({
             fontSize={fontSize}
           />
         ) : (
-          <Text variant="cell" fontSize={fontSize}>
-            {" "}
-            {storyLine.translatedContent!!}{" "}
-          </Text>
+          <Tooltip hasArrow label={tooltipCopy}>
```

Comment:
hmmmm now the reviewer also sees this tooltip, which is unnecessary. Can you modify the TranslationTable to be able to 
- have disabled cells and no tool tip
- have disabled cells and tool tip

Reply:
eg http://localhost:3000/review/6/13 shows tool tips